vpnbook

```diff
@@ -212,60 +272,62 @@ then
 else
 	mkdir -p ${vpnbook_folder} 	
 	
-	for i in PL226 DE4 US1 US2 CA222 FR1
+	for i in pl134 de20 us1 uk205 ca149 fr200 
 	do
 		__zip_downloader ${i}
 	done
 	 	
 	echo "Password retrieved from server: ${pwd}"
-	# Check if the file with Username and Password exists, if so, delete it.
+
 	if [ -f ${vpnbook_folder}/login.conf ]
 	then
 		rm ${vpnbook_folder}/login.conf
 	fi
 
-	# Create a new file to store the Username and Password
+
 	echo "vpnbook" >> ${vpnbook_folder}/login.conf
 	echo $pwd >> ${vpnbook_folder}/login.conf
 
-	# Test code to verify command line processing
+
 	if [ ${interactive} = "1" ]; then
 		__show_menu
 		read opt
 		
 		case $opt in
-		1) var=PL226;;
-		2) var=DE4;;
-		3) var=US1;;
-		4) var=US2;;
-		5) var=CA222;;
-		6) var=FR1;;
+		1) var=pl134;;
+		2) var=de20;;
+		3) var=us1;;
+		4) var=uk205;;
+		5) var=ca149;;
+		6) var=fr200;;
 		*) echo "Exit"; exit;;
 		esac
 	else
 		var=${server}
 	fi
 
+__check_if_up ${vpnbook_if_name}
+if [ $? -eq 1 ]; then
+    echo "VPN Tunnel already established. Nothing to do."
+    exit 0
+fi
 	echo "Starting the Vpn Tunnel"
-	# Login to sudo	
+	
 	sudo -v
 	
 	# Start the connection and if whatever error occurs, then connect to the first available server
 	__connect_to_first_available ${var}
 	
 	# Check if the given interface is up and running
-	__check_if_up ${vpnbook_if_name}
+	# __check_if_up ${vpnbook_if_name}
```

Comment:
I think this line might be removed

Reply:
Ig its required to check if the connection is really established or its just gimick
to remove it ig we need to add a check with in manual password entry function